tests/actor/test_battery_status.py

```diff
@@ -538,7 +538,7 @@ async def test_sync_blocking_interrupted_with_invalid_message(
         assert tracker._get_new_status_if_changed() is None
 
         tracker._handle_status_set_power_result(
-            SetPowerResult(succeeded={BATTERY_ID}, failed={})
+            SetPowerResult(succeeded={BATTERY_ID}, failed=set())
```

Comment:
Ha! This change uncovered a bug-ish (I mean, it was not a bug because it was returning an iterable, but still).

Reply:
yes, the great traps in python. :-/